### PR TITLE
APIGOV-21967 Add Product Foundry section

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add section

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier @robertengaxway Please review and add content. Thanks!

**Comment by lbadenhop** [comment]
@robertengaxway @cbordier Integrate with API / CLI topic created.

**Comment by lbadenhop** [comment]
@DesertLeprechaun Can you please provide content for the "Integrate with API / CLI" topic in this pull request? If you aren't able to provide the content, please let me know. Thanks!

File: content/en/docs/manage_product_foundry/foundry_integrate_cli_api.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+    }
+}
+```
+
+Your product is now available in the Marketplace for API consumers to subscribe to. You can see your Marketplace at the domain configured in your [Marketplace settings page](https://platform.axway.com/org/marketplace/settings).
+
+## Remove a product
+
+Before a product can be deleted its state must first be marked as `archived`.
+
```
Comment (cbordier) [inline content/en/docs/manage_product_foundry/foundry_integrate_cli_api.md:432]: Before a product can be deleted, its state must first be marked as `deprecated` and then `archived`

**Review by cbordier — COMMENTED** [review]
@lbadenhop, I reviewed the content.
I also updated the management page content.
Can you have a look.

**Review by cbordier — APPROVED** [review]
I approve the changes.

**Pull request merged**